src/main/scala/nl.knaw.dans.easy.sword2/DepositProperties.scala

```diff
@@ -84,6 +84,11 @@ class DepositProperties(depositId: DepositId, depositorId: Option[String] = None
     this
   }
 
+  def setDepositSource(source: String): Try[DepositProperties] = Try {
+    properties.setProperty("deposit.source", source)
```

Comment:
Following the comment in https://github.com/DANS-KNAW/easy-specs/pull/82#discussion_r310344792, please rename this property to `deposit.origin`. Also rename the parameter name and method name accordingly.